assignments/this.js

```diff
@@ -1,26 +1,51 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global Biding. In this instance the 'this' will refer to whatever window you are using. 
+* 2. Implicit Binding. In this instance the 'this' is implied. 
+    When calling a function with a . 'this' is whatever is left of the dot. 
+* 3. New Binding. When a constructor is used, the 'this' is the NEW object. 
+* 4. Explicit Binding. This is when you use a build in JS function like call() or apply() that is 
+    specifically created to tell 'this' what to be. 
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
 // code example for Window Binding
+console.log(this);
 
 // Principle 2
 
 // code example for Implicit Binding
+const person = {
+    name: 'Chaz',
+    age: 30,
+    occupation: 'programmer',
+    introduce: function(school) {
```

Comment:
❓ What argument is the `school` parameter accepting? It is not being used within the method neither.